### ISSUE TITLE
`sshcode --version` does not print version info

### ISSUE THREAD

**Issue by samuela** [cdr/sshcode#154]
I installed sshcode via `go get -u go.coder.com/sshcode`, and when I run `sshcode --version` I don't get any version info:
```
pi@raspberrypi:~ $ sshcode --version

pi@raspberrypi:~ $ 
```

**Comment by fonnesbeck** [comment]
Confirming this on Debian Linux, using sshcode built from source.

**Comment by Merith-TK** [comment]
this is probably because those don't have release info?

But can confirm this does happen on termux, Android 8.1

**Comment by IngCr3at1on** [comment]
Yeah this is an issue with fetching sshcode using `go get` instead of downloading a release binary. Since we're not running `ci/build.sh` the version information is never set. Off the top of my head I don't think there's an easy way to make this fill in automatically when running `go get` and the obvious intention is to not have to update the version manually... I wonder if instead of printing an empty string it should just say something like `unofficial`?

**Comment by Merith-TK** [comment]
Maybe git version?

**Comment by IngCr3at1on** [comment]
How do you get the git version without a script? There's no way (to my knowledge) to make get such a value using go get to build the binary.

**Comment by Merith-TK** [comment]
uhm, check the git version of `GOPATH/src/github.com/cdr/sshcode`?

**Comment by IngCr3at1on** [comment]
The issue is that running `go get -u go.coder.com/sshcode` is leveraging the go tooling, it sees the presence of a main.go and compiles this to a binary. There's no way to make that tooling find the version and add it to the version string. The string value itself is set at compile-time using a ldflag which you otherwise don't get to pass in when running go get.

**Comment by deansheather** [comment]
If you're installing from source there's no release version built into the binary. I don't think it's a big deal that it prints nothing in this case, but I'll accept a PR so custom builds say `unofficial`.

**Comment by IngCr3at1on** [comment]
> If you're installing from source there's no release version built into the binary. I don't think it's a big deal that it prints nothing in this case, but I'll accept a PR so custom builds say `unofficial`.

Yeah personally I'm totally indifferent my comments and suggestions here were just to other's benefit. That said if it's wanted by others I'll gladly submit such a PR, another option might be to simply link this issue from a FAQ in the readme, or yes leave it entirely as is as it's a pretty small non-annoyance :smile:

**Comment by Merith-TK** [comment]
One way to build with git versions is to use a script to sed a go file that
contained a variable?
Contents would be

package main
const version = "Version/commit string"

**Comment by IngCr3at1on** [comment]
> One way to build with git versions is to use a script to sed a go file that contained a variable? Contents would be package main const version = "Version/commit string"

The problem is that script will _never_ be called by `go get` which is _always_ going to be the preferred method for installing a go binary directly from source.

**Comment by Merith-TK** [comment]
Ah, yeah, go really needs to get make file support, like just run `make go`
if go exists in the make file. And it would be steps done before "go build"
